discord.d.ts

```diff
@@ -663,10 +663,10 @@ declare module "discord.js" {
         deleteAll(): Promise<void[]>;
         every(fn: Function, thisArg?: Object): boolean;
         exists(prop: string, value: any): boolean;
-        filter(fn: Function, thisArg?: Object): Collection<any, any>;
-        find(propOrFn: string | Function, value?: any): any;
-        findAll(prop: string, value: any): any[];
-        findKey(propOrFn: string | Function, value?: any): any;
+        filter(fn: Function, thisArg?: Object): Collection<key, value>;
+        find(propOrFn: string | Function, value?: any): value;
```

Comment:
`find` falls along the same category. The function used in find must return a boolean/truthy value and will return the first item in the Collection that leads to the functions' expression evaluating to true/truthy. It will not return anything of a type other than those contained within the Collection.